saleor/cart/migrations/0002_auto_20170206_0407.py

```diff
@@ -41,7 +41,7 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='cart',
             name='total',
-            field=django_prices.models.PriceField(currency='USD', decimal_places=2, default=0, max_digits=12, verbose_name='total'),
```

Comment:
We should not modify preexisting migrations in any way. If you want to change something, please create new migration instead of rewriting history.

Reply:
We have to since the API of `django-prices` has changed and without adjusting the old migrations, they won't work at all (signature of `PriceField` is different).